OpenAI-DotNet/Chat/ChatRequest.cs

```diff
@@ -68,6 +69,8 @@ public sealed class ChatRequest
         /// </param>
         public ChatRequest(
             IEnumerable<Message> messages,
+            IEnumerable<Function> functions = null,
```

Comment:
Let's put this at the end of the method signature so that it's not a braking change

Reply:
I hate it, but ok.